src/classic_movie.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file classic_movie.h
+ * @author Austin Arlitt
+ */
+
+#ifndef CLASSIC_MOVIE_H
+#define CLASSIC_MOVIE_H
+
+// includes TBD
+using namespace std;
+
+/**
+ * A class for classic movies in the rental movie store
+ */
+class ClassicMovie : public Movie {
+public:
+    ClassicMovie(string, string, string, int, int, string);     // constructor
+    int compare(const Movie &);                         // compares two movies
```

Comment:
Consider adding the `override` keyword for any concrete implementations of virtual methods. This will allow the compiler to catch mismatched names or type signatures. http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#Rh-override